misk-metrics/src/main/kotlin/misk/metrics/MetricsModule.kt

```diff
@@ -9,7 +9,9 @@ import misk.inject.asSingleton
 class MetricsModule : KAbstractModule() {
   override fun configure() {
     bind<CollectorRegistry>().toProvider(CollectorRegistryProvider::class.java).asSingleton()
-
+    bind<HistogramRegistry>().toProvider(HistogramRegistryProvider::class.java).asSingleton()
```

Comment:
Is there a `.toProvider<T>()` method we could use instead?

Reply:
I just copied what was here previously. It was grosser than I'd have liked. How about I look into it for a follow on pr when i'm back making changes? I think ultimately this module will not be necessary, or very, very simple.